### PR TITLE
Make indented syntax available for SASS via the "sass" file extension

### PR DESCRIPTION
**Pull request by ErichDonGubler**
Add support for [indented syntax in SASS](http://sass-lang.com/documentation/file.INDENTED_SYNTAX.html) via the `sass` file extension (instead of `scss`) for files inside the `sass` folder.

### CODE REVIEW

File: components/site/src/lib.rs [hunk 1]
```diff
@@ -537,17 +537,34 @@ impl Site {
...
             .filter_map(|e| e.ok())
             .filter(|entry| !entry.as_path().file_name().unwrap().to_string_lossy().starts_with('_'))
             .collect::<Vec<_>>();
 
-        let mut sass_options = Options::default();
-        sass_options.output_style = OutputStyle::Compressed;
+        let mut compiled_paths = Vec::new();
         for file in files {
-            let css = compile_file(&file, sass_options.clone())?;
+            let css = compile_file(&file, options.clone())?;
```
Comment (ErichDonGubler) [inline components/site/src/lib.rs]: Whoops, just noticed an extraneous `clone` here.

File: components/site/src/lib.rs [hunk 2]
```diff
@@ -537,17 +537,34 @@ impl Site {
...
+        options.output_style = OutputStyle::Compressed;
+        let mut compiled_paths = self.compile_sass_glob(&sass_path, "scss", options.clone())?;
+
+        options.indented_syntax = true;
+        compiled_paths.extend(self.compile_sass_glob(&sass_path, "sass", options)?);
+
+        compiled_paths.sort();
+        for window in compiled_paths.windows(2) {
+            if window[0].1 == window[1].1 {
+                bail!("SASS path conflict: \"{}\" and \"{}\" both compile to \"{}\"", window[0].0.display(), window[1].0.display(), window[0].1.display());
```
Comment (Keats) [inline components/site/src/lib.rs]: A bit too long, can you break it like

```
bail!(
  "msg",
   one_arg,
   etc,
)
```
?

File: components/site/src/lib.rs [hunk 3]
```diff
@@ -537,17 +537,34 @@ impl Site {
             sass_path
         };
 
-        let sass_glob = format!("{}/**/*.scss", sass_path.display());
-        let files = glob(&sass_glob)
+        let mut options = Options::default();
```
Comment (Keats) [inline components/site/src/lib.rs]: can you also rename the import to be `SassOptions`? Now that it is passed around, seeing just `Options` as type is a bit weird

**Comment by ErichDonGubler** [comment]
Done. :) Let me know if there's anything else you'd like me to polish!

**Comment by Keats** [comment]
Thanks!

**Pull request merged**